smartcar-auth/src/main/java/com/smartcar/sdk/Helper.java

```diff
@@ -65,4 +65,14 @@ protected static String arrayToString(String[] array) {
         }
         return stringBuilder.toString();
     }
+
+    protected static VehicleInfo makeFullVehicle(String vin, String make, String model, Integer year) {
+        VehicleInfo vehicle = new VehicleInfo.Builder()
+                .setVin(vin)
+                .setMake(make)
+                .setModel(model)
```

Comment:
Does it matter here that `model` is sometimes null?

Reply:
The attribute will just be null, I can test it out.

Reply:
Since Strings are a class, the model attribute will just get to null and nothing will break.